sal_interfaces/HVAC/HVAC_Telemetry.xml

```diff
@@ -1,256 +1,1010 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version='1.0' encoding='UTF-8'?>
 <?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALTelemetrySet.xsl"?>
-<SALTelemetrySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALTelemetrySet.xsd">
-<SALTelemetry>
+<SALTelemetrySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALTelemetrySet.xsd">
+  <SALTelemetry>
     <Subsystem>HVAC</Subsystem>
-    <Version>3.8</Version>
-    <Author></Author>
-    <EFDB_Topic>HVAC_lowerAHUStatus</EFDB_Topic>
+    <EFDB_Topic>HVAC_topic_(mqtt)_(todas_variables_)</EFDB_Topic>
```

Comment:
The parenthesis are not allowed.

Note: I encourage you to run the SAL IDL generator on these files, as well as run the unit tests. In the development Docker image run `make_idl_file.py HVAC`. It is even more important for that to run correctly (and generate usable IDL files) that it is for the ts_xml unit tests to run, since it is the files generated by SAL that we actually use.

Reply:
That one slipped my attention. I improved my code because this one is not supposed to be in the XML. Also I improved the TopicNaming test to catch this kind of naming errors.